profiles/common/modules/custom/ecas/includes/ecas.inc

```diff
@@ -213,7 +213,7 @@ function ecas_login_page() {
 function ecas_logout($full_logout = FALSE) {
   global $user;
 
-  watchdog('user', 'Session closed for %name.', array('%name' => theme('placeholder', $user->name)));
+  watchdog('user', 'Session closed for %name.', array('%name' => $user->name));
```

Comment:
I think it's better to use the API function: `format_username()`.

Another PR is here about it: https://github.com/ec-europa/platform-dev/pull/1200